contracts/protocol/PositionManager.sol

```diff
@@ -228,17 +228,17 @@ contract PositionManager is
     function cancelLimitOrder(uint128 pip, uint64 orderId)
         external
         onlyCounterParty
-        returns (uint256 size, uint256 partialFilled)
+        returns (uint256 remainingSize, uint256 partialFilled)
     {
-        (size, partialFilled) = tickPosition[pip].cancelLimitOrder(orderId);
+        (remainingSize, partialFilled) = tickPosition[pip].cancelLimitOrder(orderId);
```

Comment:
in the formula, this variable is calculated by totalLiquidityOfPip - partialFilled so it is equal to remaining liquidity that was not filled in that pip. the name size does not describe the formula enough